junitparser/junitparser.py

```diff
@@ -480,6 +521,8 @@ class TestCase(Element):
     _tag = "testcase"
     name = Attr()
     classname = Attr()
+    file = Attr()
```

Comment:
I would like to keep just the attributes in the original spec in http://windyroad.com.au/2011/02/07/apache-ant-junit-xml-schema/ , so I don't think they belong here.
It would be better if we can use preset schema for each flavor though.

Reply:
I could inject a function or a class like this `JUnitXml.fromfile(filepath, flavor)` where flavor would add the support for custom attributes.
I'll have to make sure that it doesn't break when you want to load multiple report of different flavors though.